community/aqua/general/amplitude_estimation.ipynb

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
The notebook is missing the standard qiskit tutorial header (image, where the latest version can be found and list of contributors)